include/drivers/bluetooth/hci_driver.h

```diff
@@ -113,6 +118,9 @@ struct bt_hci_driver {
 	/** Bus of the transport (BT_HCI_DRIVER_BUS_*) */
 	enum bt_hci_driver_bus bus;
 
+	/** Specific controller quirks. */
```

Comment:
We should expand the documentation here by stating that the driver might set those bits. Also, are they going to be bidirectional? Can the Host set a bit so that the controller acts differently? This should be clarified as well

Reply:
I've updated the code comment to clarify this